geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
+	txSendAsync := `
+		_status_catalog.commands.send({
+			"from": "` + TestConfig.Account1.Address + `",
+			"to": "` + TestConfig.Account2.Address + `",
+			"value": "0.000001"
+		})
+	`
+	rxSendJS := LoadFromFile("../../static/testdata/jail/tx-send/rx-send.js")
+	require.NotEmpty(rxSendJS, "Base script is empty")
+
+	s.StartTestNode(params.RopstenNetworkID)
+	defer s.StopTestNode()
```

Comment:
If the node is not started jail will return an error `node has stopped/not started`.